sandbox/components/SandboxSectionComponent.vue

```diff
@@ -77,14 +77,22 @@ const hasDescription = computed((): boolean => !!(props.description || slots.des
       font-size: $kui-font-size-40;
       line-height: $kui-line-height-40;
       margin: $kui-space-0;
-      max-width: 700px
+      max-width: 700px;
```

Comment:
nit: can we use our breakpoint tokens here?

Reply:
Done